### ISSUE TITLE
Anyone can make a good public client downloable? actually all clients are basic

### ISSUE THREAD

**Issue by liomessip** [Barbosik/MultiOgar#142]

**Comment by DTMClanDeveloper** [comment]
yeah that would be a good idea, like gota.io a downloadable client would get many more peoples attention.

**Comment by acydwarp** [comment]
Currently working on a client that is fully compatible with MultiOgar servers. It performs extremely well.

Preview: https://www.youtube.com/watch?v=Mwfjz2c8LSs

Client download will also soon be available for all platforms.

**Comment by DTMClanDeveloper** [comment]
acyd firstly, i love you and im a huge fan

secondly, astrio isnt working :( and ive been wathcing zt since 1k subs btw

**Comment by NatsuTheGreat** [comment]
I want to try astr.io

**Comment by DTMClanDeveloper** [comment]
i do too but it wont work.

**Comment by acydwarp** [comment]
I'm in the process of adding some important client features. There will be a beta round this weekend. I posted a timer on the website.

**Comment by DTMClanDeveloper** [comment]
ACYD I LOVE YOU AND ZT.

**Comment by David1ali12** [comment]
Looks same as agarplus

**Comment by liomessip** [comment]
@acydwarp , 
When i can download your client?

**Comment by fayizan** [comment]
@acydwarp @David1ali12 @NatsuTheGreat @Barbosik could someone please create a separate server tracker website for MULTIOGAR...it will be really cool..!!

**Comment by Barbosik** [comment]
**fayizan**: yes, it will be added soon. It also will include web service which will allows clients to request server list in JSON.

**Comment by fayizan** [comment]
@Barbosik Cool :+1:

**Comment by lulzyyy** [comment]
@fayizan I created server tracker for MultiOgar http://c0nsume.me/tracker.php

**Comment by fayizan** [comment]
@lulzyyy Cool I really appreciate..!!,can u please add more better GUI

**Comment by nulcearc** [comment]
@acydwarp I'm also working on my client but i want it to work with official servers

**Comment by MastaCoder** [comment]
You can't get them to work with official servers, they disallow connections from anything but agar.io. (It was localhost too but don't think that works anymore.)

**Comment by Barbosik** [comment]
current trackers very slow and slow down the server. I implemented more fast tracker. Currently working on UI for it

**Comment by SimonOrJ** [comment]
@Barbosik: I was trying to implement my own version of server trackers, but I wasn't sure how practical it would be with all the other server trackers that track the same servers...

If you're working on the GUI for browser access, I would like to lend a hand.

**Comment by nulcearc** [comment]
@MastaCoder no sometime i connected to server and got world size

**Comment by liomessip** [comment]
@acydwarp  El cliente donde es descargable?

**Comment by CalvinDKHD** [comment]
@acydwarp BIG FAN! making a game client will be agarplus, but its hard to edit.... its up atm - cellextreme.pw     please add me on skype - calvin.leonhardt

btw on the client i have the chat doesnt work! so if you can help me fix that thats good! ill even pay a little!

THANKS

**Comment by The-Lighting** [comment]
Does anyone got the astr.io client download??

**Comment by Megabyte918** [comment]
Why do you need to download it?

**Comment by The-Lighting** [comment]
I need it because i will also self host my own agar server.. Currently: http://www.okj-server.tk  and i will also credit acydwarp for that.. and that client looks cool and i really need it... so i can edit it .. and make it connect to my servers.

**Comment by Mistik** [comment]
Also try https://senpa.io

**Comment by Mistik** [comment]
> Currently working on a client that is fully compatible with MultiOgar servers. It performs extremely well.
> 
> Preview: https://www.youtube.com/watch?v=Mwfjz2c8LSs
> 
> Client download will also soon be available for all platforms.

where is the download
